### PR TITLE
Update peerDependencies to include React 18

### PR DESCRIPTION
**Pull request by timarney**
Running into  dependency issues when using React 18

Screen Shot 2022-04-13 at 10 17 10 AM

If I force install importing is working as expected.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,8 +102,8 @@
     "webpack-dev-server": "^3.7.0"
   },
   "peerDependencies": {
-    "react": "^16.8.0 || ^17.0.0",
-    "react-dom": "^16.8.0 || ^17.0.0"
+    "react": "^16.8.0 || ^18.0.0",
```
Comment (unframework) [inline package.json:105]:
```suggestion
    "react": "^16.8.0 || >=17.0.0",
```

File: package.json [hunk 2]
```diff
@@ -102,8 +102,8 @@
...
+    "react-dom": "^16.8.0 || ^18.0.0"
```
Comment (unframework) [inline package.json:106]:
```suggestion
    "react-dom": "^16.8.0 || >=17.0.0"
```

**Review by unframework — CHANGES_REQUESTED** [review]
Hi Tim, thanks for fixing this. I suggested a couple tweaks to ensure that projects with React v17 are not affected, please see review comments. Thanks!

**Comment by timarney** [comment]
Perfect that makes sense.

**Review by unframework — APPROVED** [review]

**Comment by unframework** [comment]
Hi Tim, I just published this in v0.7.1, let me know if there are any issues, thanks again for contributing!

**Pull request merged**